src/test/java/study/racing/domain/CarsTest.java

```diff
@@ -0,0 +1,50 @@
+package study.racing.domain;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class CarsTest {
+
+    @Test
+    void 객체_정상생성_성공() {
+        Cars raceAttempt = new Cars("car1", "car2");
+        assertThat(raceAttempt.getCars().size()).isEqualTo(2);
+        assertThat(raceAttempt.getCars().get(0).getName()).isEqualTo(new CarName("car1"));
+        assertThat(raceAttempt.getCars().get(1).getName()).isEqualTo(new CarName("car2"));
```

Comment:
```suggestion
        assertThat(raceAttempt.getCars()).contains(new Car("car1"). new Car("car2"));
```
가능하면 테스트 코드의 assert 또한 객체의 값을 꺼내기보다(get) 위와 같이 객체 단위로 비교하도록 접근해 보면 어떨까?